run_robot_test.sh

```diff
@@ -149,6 +150,12 @@ while [ "$#" -gt 0 ]; do
       shift
       ;;
 
+    --dryrun_action)
+      shift
+      DRYRUN_ACTION=$1
+      shift
+      ;;
```

Comment:
is it really needed? We could use the --extra-robot-args '--dryrun'

Reply:
This is different from the --dryrun switch because I use it to force the log files in the top level folder instead of a subfolder, otherwise it becomes impossible to parse the results and post a comment through the action